examples/apiexport/main.go

```diff
@@ -121,7 +121,14 @@ func main() {
 					return reconcile.Result{}, fmt.Errorf("failed to get configmap: %w", err)
 				}
 
-				log.Info("Reconciling configmap", "name", s.Name, "uuid", s.UID)
+				log.Info("Reconciling ConfigMap", "name", s.Name, "uuid", s.UID)
+
+				secrets := &corev1.SecretList{}
+				if err := client.List(ctx, secrets); err != nil {
+					return reconcile.Result{}, err
+				}
+
+				log.Info("Found Secret objects in Workspace of ConfigMap", "count", len(secrets.Items))
```

Comment:
I don't like this here in the example. We need e2e tests.

Reply:
I agree. I can remove it from the example, I just needed a way to validate it. With the sdk changes to include a testing framework, we can write e2e tests.

Reply:
@sttts removed the example code changes.